bp_be/src/include/bp_be_dcache/bp_be_dcache_pkg.vh

```diff
@@ -54,8 +55,24 @@ package bp_be_dcache_pkg;
     
   } bp_be_dcache_lce_data_mem_opcode_e;
 
+
+  typedef enum logic [1:0] {
```

Comment:
Perhaps this whole package/header should be consolidated into a single cache package. The I$ would then, naturally, support only a subset of operations (i.e., it would not support stores).

Reply:
agreed